draft-packages/button/KaizenDraft/Button/components/GenericButton.tsx

```diff
@@ -0,0 +1,175 @@
+import { Icon } from "@kaizen/component-library"
+import classNames from "classnames"
+import * as React from "react"
+
+const styles = require("./GenericButton.module.scss")
+
+type GenericProps = {
+  id?: string
+  label: string
+  destructive?: boolean
+  disabled?: boolean
+  form?: boolean
+  reversed?: boolean
+  icon?: React.SVGAttributes<SVGSymbolElement>
+  onClick?: (e: MouseEvent) => void
+  href?: string
+  newTabAndIUnderstandTheAccessibilityImplications?: boolean
+  type?: "submit" | "reset" | "button"
+  automationId?: string
+  fullWidth?: boolean
+  disableTabFocusAndIUnderstandTheAccessibilityImplications?: boolean
+  analytics?: Analytics
+}
+
+type LabelProps = {
+  iconPosition?: "start" | "end"
+  primary?: boolean
+  secondary?: boolean
+  reverseColor?: "lapis" | "ocean" | "peach" | "seedling" | "wisteria" | "yuzu"
```

Comment:
Are we in a position to deprecate lapis here?

Reply:
Oh good call! Yes we can, looks like it was just using Wisteria anyway: https://github.com/cultureamp/kaizen-design-system/pull/460/files?file-filters%5B%5D=.json&file-filters%5B%5D=.scss&file-filters%5B%5D=.ts&file-filters%5B%5D=.tsx#diff-3b161e88246c9ecb44a018720c570bc3R247

I'll do that in a sec.